website/src/components/articles/article-sections.tsx

```diff
@@ -111,47 +118,38 @@ const Title = styled.h6`
   }
 `;
 
-const Content = styled.div`
-  ul {
-    display: flex;
-    flex-direction: column;
-    margin: 0;
-    padding: 0 25px 10px;
-    list-style-type: none;
-
-    li {
-      flex: 0 0 auto;
-      margin: 5px 0;
-      padding: 0;
-      line-height: initial;
-
-      > p {
-        margin: 0;
-        padding: 0;
-      }
-
-      > ul {
-        padding-right: 0;
-      }
-
-      &.active > a,
-      > p.active > a {
-        font-weight: bold;
-      }
-
-      > a,
-      > p > a {
-        font-size: 0.833em;
-        color: #666;
-
-        :hover {
-          color: #000;
-        }
-      }
-    }
-
-    @media only screen and (min-width: 1320px) {
-      padding: 0 20px 10px;
-    }
+const TocItemContainer = styled.ul`
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  padding: 0 25px 10px;
+  list-style-type: none;
+
+  @media only screen and (min-width: 1320px) {
```

Comment:
Why not using the helper from Fred?

Reply:
Not sure if there is a helper for `1320px` but maybe you could check this out :-)

Reply:
Does not exist.

The breakpoints and helper can be improved. In my opinion they should be degrading and either desktop- or mobile first.
The currently used breakpoints are a wild mix between  `min-width` and `max-width`.